coldfront/core/group/management/commands/create_default_user_groups.py

```diff
@@ -0,0 +1,119 @@
+    {"id": 220, "codename": "delete_allocationusernote"},
+    {"id": 221, "codename": "view_allocationusernote"},
+    {"id": 222, "codename": "add_allocationuser"},
+    {"id": 223, "codename": "change_allocationuser"},
+    {"id": 224, "codename": "delete_allocationuser"},
+    {"id": 225, "codename": "view_allocationuser"},
+    {"id": 226, "codename": "add_allocationadminnote"},
+    {"id": 227, "codename": "change_allocationadminnote"},
+    {"id": 228, "codename": "delete_allocationadminnote"},
+    {"id": 229, "codename": "view_allocationadminnote"},
+    {"id": 230, "codename": "add_allocationaccount"},
+    {"id": 231, "codename": "change_allocationaccount"},
+    {"id": 232, "codename": "delete_allocationaccount"},
+    {"id": 233, "codename": "view_allocationaccount"},
+    {"id": 234, "codename": "add_allocationchangerequest"},
+    {"id": 235, "codename": "change_allocationchangerequest"},
+    {"id": 236, "codename": "delete_allocationchangerequest"},
+    {"id": 237, "codename": "view_allocationchangerequest"},
+    {"id": 238, "codename": "add_allocationchangestatuschoice"},
+    {"id": 239, "codename": "change_allocationchangestatuschoice"},
+    {"id": 240, "codename": "delete_allocationchangestatuschoice"},
+    {"id": 241, "codename": "view_allocationchangestatuschoice"},
+    {"id": 242, "codename": "add_historicalallocationchangerequest"},
+    {"id": 243, "codename": "change_historicalallocationchangerequest"},
+    {"id": 244, "codename": "delete_historicalallocationchangerequest"},
+    {"id": 245, "codename": "view_historicalallocationchangerequest"},
+    {"id": 246, "codename": "add_historicalallocationattributechangerequest"},
+    {"id": 247, "codename": "change_historicalallocationattributechangerequest"},
+    {"id": 248, "codename": "delete_historicalallocationattributechangerequest"},
+    {"id": 249, "codename": "view_historicalallocationattributechangerequest"},
+    {"id": 250, "codename": "add_allocationattributechangerequest"},
+    {"id": 251, "codename": "change_allocationattributechangerequest"},
+    {"id": 252, "codename": "delete_allocationattributechangerequest"},
+    {"id": 253, "codename": "view_allocationattributechangerequest"},
+]
+DEFAULT_RIS_USER_GROUPS = [
+    {
+        "name": "RIS-UserSupport",
+        "permissions": DEFAULT_RIS_USERSUPPORT_GROUP_PERMISSIONS,
+    }
+]
+
+
+class Command(BaseCommand):
+    """
+    Command to create default user groups.
+    """
+
+    help = "Create default user groups"
+
+    def handle(self, *args, **options):
+        print("Creating default user groups ...")
+        for group in DEFAULT_RIS_USER_GROUPS:
+            print(f"Creating group: {group['name']}")
+            Group.objects.get_or_create(name=group["name"])
+            for permission in group["permissions"]:
+                print(f"Adding permission: {permission['codename']}")
+                group_obj = Group.objects.get(name=group["name"])
```

Comment:
`get_or_create` returns a tuple containing the `Group` object, so you don't necessarily need the repeated `get` call.